src/db/rss-parser.ts

```diff
@@ -16,14 +17,22 @@ const getRecentArticles = async (): Promise<Article[]> => {
     return parser.parseURL(articles);
   });
 
+  const parseImage = function (content) {
+    const $ = cheerio.load(content);
+    const img = $('img').attr('src');
```

Comment:
I've never seen `$` used for naming const/vars/funcs before, is this common practice for using cheerio or typescript? Just seems unfamiliar and not that expressive?

Reply:
I think this is how cheerio recommends using it just because it mocks jQuery with line 22 being the same between both cheerio and jQuery. This is syntax is also used with pear-backend and getting the majors in Web-Scraper.ts